### ISSUE TITLE
Tooltip not respecting a on-mouse hover for accessibility 

### ISSUE THREAD

**Issue by crutkas** [microsoft/microsoft-ui-xaml#3521]
**Describe the bug**

**Steps to reproduce the bug**

1. Go to tooltip
2. hover so it becomes visible
3. move mouse on it

**Expected behavior**

according to https://w3c.github.io/wcag21/understanding/content-on-hover-or-focus.html 2nd bullet, it is supposed to not be dismissed

This was flagged in PowerToy's accessibility pass.  https://github.com/microsoft/PowerToys/issues/7058

**Screenshots**

![tooltipbehavior](https://user-images.githubusercontent.com/9866362/97011884-8600e900-1547-11eb-983c-232e76608505.gif)

**Version Info**

Xaml Gallery 1.2.16.0

NuGet package version: 
2.5.0-prerelease.200923002 is what is being used inside PowerToys

Windows app type:
| UWP              | Win32            |
| :--------------- | :--------------- |
| Yes | Yes |

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2020 Update (19041)            | Yes |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :----------------- | :--------------- |
| Desktop            | Yes |
| Xbox               |  |
| Surface Hub        |  |
| IoT                |  |

**Additional context**

**Comment by ranjeshj** [comment]
This is a known accessibility bug tracked internally [here](https://microsoft.visualstudio.com/OS/_workitems/edit/24000491).

**Comment by YuliKl** [comment]
Closing as a dupe.

**Comment by crutkas** [comment]
This isn't a dup, this is an issue that should be tracked.  Just because it is internally tracked doesn't mean it shouldn't be externally.

**Comment by crutkas** [comment]
Please reopen

**Comment by Felix-Dev** [comment]
To add to what @crutkas said, tracking issues externally (even though they might already be tracked internally) will allow the community to contribute fixes if desired by the team.

**Comment by ranjeshj** [comment]
Leaving it open for tracking in github.
